plugin.xml

```diff
@@ -33,8 +34,9 @@
                 <source url="https://github.com/CocoaPods/Specs.git"/>
             </config>
             <pods use-frameworks="true">
-                <pod name="PurchasesHybridCommon" spec="1.6.2"/>
+                <pod name="PurchasesHybridCommon" spec="1.7.1"/>
             </pods>
         </podspec>
     </platform>
+    <params name="ANDROIDXENABLED" value="true" />
```

Comment:
ooohhh nice, I hadn't seen this before and I'd always enable it manually per-app